src/app/checkboxes/checkboxes.demo.html

```diff
@@ -49,21 +49,55 @@
             <label>Indeterminate Checkbox</label>
             <div class="checkbox-inline">
                 <clr-checkbox type="checkbox" id="checkbox_9" [(clrIndeterminate)]="indeterminateState">
-                    Indeterminate Checkbox.
+                    Clarity indeterminate checkbox
                 </clr-checkbox>
                 <div>
-                    <div>
-                        <b>NOTE:</b> Native checkbox elements must set indeterminate state with JavaScript.
-                    </div>
                     <button class="btn btn-sm" (click)="onToggleIndeterminateState($event)" type="button">
                         Toggle Indeterminate State
                     </button>
                 </div>
+                <div class="checkbox-inline">
+                    <input type="checkbox"
+                           id="checkrads_10"
+                           [indeterminate]="nativeIndeterminateState">
+                    <label for="checkrads_10">Native indeterminate checkbox</label>
+                    <div>
+                        <b>NOTE:</b> Native checkbox elements must set indeterminate state with JavaScript.
+                    </div>
+                </div>
             </div>
         </div>
     </section>
 </form>
 
+<form #form="ngForm" novalidate>
+    <section class="form-block">
+        <div class="form-group">
+            <label>With ngForm & ngModel</label>
+            <clr-checkbox
+                    name="termsCheckbox"
+                    class="checkbox-inline"
+                    id="checkrads_11"
+                    [clrInline]="true"
+                    [(ngModel)]="termsAgreement">
+                I agree to the terms and conditions. {{ form.value | json }}
+            </clr-checkbox>
+        </div>
+    </section>
+</form>
+
+<form>
+    <section class="form-block">
+        <div class="form-group">
+            <label>With a list of objects</label>
+            <clr-checkbox *ngFor="let item of list"
+                          [(clrChecked)]="item.running"
+                          [clrDisabled]="item.disabled">
+                {{ item.name }} at {{ item.ip }} {{ item.disabled ? ' Is disabled.' : 'Is ok.' }}
```

Comment:
To make the example more relevant,  you should change this to:
```
{{ item.disabled ? ' is disabled.' : (item.running ? 'is running.': 'is off.') }}
```
Or something similar that shows how checking a box actually modifies the model through `[(clrChecked)]`.